src/for-students-FAQ/for-students-FAQ.js

```diff
@@ -0,0 +1,37 @@
+import { Accordion, Container, Title, Divider, Image } from "@mantine/core";
+import "./for-students-FAQ.css";
+
+function FaqCard({ faq_info }) {
+  const items = faq_info.map((faq, index) => (
+    <Accordion.Item key={index} value={faq.question}>
+      <Accordion.Control className="question">
+        <b>{faq.question}</b>
+      </Accordion.Control>
+
+      <Accordion.Panel>{faq.answer}</Accordion.Panel>
+    </Accordion.Item>
+  ));
+
+  return (
+    <Container fluid>
+      <Title>FAQ</Title>
+
+      <Divider my="lg"></Divider>
+
+      <p>
+        Here are some frequently asked questions that we’ve compiled! If you
+        have further questions, please contact us on our Instagram or
+        contact@uoftblueprint.org.
+      </p>
+
+      <Accordion
+        variant="separated"
+        chevron={<Image src="chevron-down.png" />}
```

Comment:
I think you might have forgotten to push the png image for the chevron? I can click open the accordian but I can't see the chevron.

Reply:
oop mb mb will push the image!